WordPress/src/main/res/layout/scan_history_activity.xml

```diff
@@ -10,20 +10,6 @@
         android:id="@+id/fragment_container_view"
         android:name="org.wordpress.android.ui.jetpack.scan.history.ScanHistoryFragment"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        app:layout_behavior="@string/appbar_scrolling_view_behavior" />
-
-    <com.google.android.material.appbar.AppBarLayout
-        android:id="@+id/appbar_main"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content">
-
-        <com.google.android.material.appbar.MaterialToolbar
-            android:id="@+id/toolbar_main"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            app:theme="@style/WordPress.ActionBar" />
-
-    </com.google.android.material.appbar.AppBarLayout>
+        android:layout_height="match_parent" />
```

Comment:
Nitpick 🔍: 1. Unused app name space can be removed
`xmlns:app="http://schemas.android.com/apk/res-auto"`

2. Do we need `CoordinatorLayout` in the activity xml now that it has been moved to the fragment?

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/13757/commits/be7f344315a44b4e2ee394d54146d98ab564ec60